### PR TITLE
[11.0] [MIG] report_qweb_element_page_visibility

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by svazquezco** [comment]
CLA -> https://github.com/odoo/odoo/pull/21709

**Comment by pedrobaeza** [comment]
Odoo CLA is different from OCA CLA.

**Comment by ghost** [comment]
Thanks for notifying @pedrobaeza! I was wrong. I have already sent the signed CLA.

File: report_qweb_element_page_visibility/views/layouts.xml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+
+    <template id="minimal_layout" inherit_id="web.minimal_layout">
+        <xpath expr="//t[@t-if='subst is True']" position="replace">
+            <t t-if="subst is True">
+                <script>
+                    function subst() {
```
Comment (jesusVMayor) [inline report_qweb_element_page_visibility/views/layouts.xml:8]: Maybe you should to ttake the subst function code from 11.0 and add the visibility code to it https://github.com/OCA/OCB/blob/11.0/addons/web/views/report_templates.xml

File: report_qweb_element_page_visibility/__init__.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# -*- coding: utf-8 -*-
```
Comment (jesusVMayor) [inline report_qweb_element_page_visibility/__init__.py]: https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-11.0

File: report_qweb_element_page_visibility/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```
Comment (jesusVMayor) [inline report_qweb_element_page_visibility/__manifest__.py]: https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-11.0

**Review by jesusVMayor — CHANGES_REQUESTED** [review]
functional testing, it works fine.

**Comment by ghost** [comment]
Thanks for review @jesusVMayor, I'll make the changes ASAP.

File: report_qweb_element_page_visibility/views/layouts.xml (hunk 1 above)
Comment (ghost) [reply report_qweb_element_page_visibility/views/layouts.xml:8]: @jesusVMayor, I think to use replace position attribute it's never a good practice but I don't know another way to do in templates. Can you help me?
Thanks!

**Comment by aitorbouzas** [comment]
@tafaRU can you give some love to this PR? I looking forward to migrate this module to 12.0

**Comment by youring** [comment]
I think this pr missed some important changes from core commit https://github.com/odoo/odoo/commit/b446930dc42479f524eab7e677c5b2007f9fa0a6#diff-e601c652af2b903bf1a699deaaad4fcb ,
and this OCA pr is also worthy to be noted here for reference /pull/247

File: report_qweb_element_page_visibility/views/layouts.xml (hunk 1 above)
Comment (youring) [reply report_qweb_element_page_visibility/views/layouts.xml:8]: @svazquezco maybe /pull/247 will give some hints, just from a user perspective view.

**Pull request closed without merging**